<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="9" passed="4" failed="5" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-05-30T14:37:22 EDT" name="Suite" finished-at="2020-05-30T14:38:54 EDT" duration-ms="92525">
    <groups>
    </groups>
    <test started-at="2020-05-30T14:37:22 EDT" name="Test" finished-at="2020-05-30T14:38:54 EDT" duration-ms="92525">
      <class name="com.hrms.testcases.LoginPageValidation">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrms.testcases.LoginPageValidation@13c10b87]" started-at="2020-05-30T14:37:22 EDT" name="setUp" finished-at="2020-05-30T14:37:24 EDT" duration-ms="2608" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validAdminLogin()[pri:0, instance:com.hrms.testcases.LoginPageValidation@13c10b87]" started-at="2020-05-30T14:37:24 EDT" name="validAdminLogin" finished-at="2020-05-30T14:37:26 EDT" duration-ms="2190" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validAdminLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrms.testcases.LoginPageValidation@13c10b87]" started-at="2020-05-30T14:37:27 EDT" name="tearDown" finished-at="2020-05-30T14:37:32 EDT" duration-ms="5115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.hrms.testcases.LoginPageValidation -->
      <class name="com.hrms.testcases.AddEmployeeValidationTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrms.testcases.AddEmployeeValidationTest@5a955565]" started-at="2020-05-30T14:38:21 EDT" name="setUp" finished-at="2020-05-30T14:38:25 EDT" duration-ms="3664" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="addEmployeeValidation()[pri:0, instance:com.hrms.testcases.AddEmployeeValidationTest@5a955565]" started-at="2020-05-30T14:38:25 EDT" name="addEmployeeValidation" finished-at="2020-05-30T14:38:33 EDT" duration-ms="8214" status="FAIL">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Keys to send should be a not null CharSequence]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Keys to send should be a not null CharSequence
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:97)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at com.sun.proxy.$Proxy12.sendKeys(Unknown Source)
at com.hrms.utils.CommonMethods.sendText(CommonMethods.java:32)
at com.hrms.testcases.AddEmployeeValidationTest.addEmployeeValidation(AddEmployeeValidationTest.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addEmployeeValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrms.testcases.AddEmployeeValidationTest@5a955565]" started-at="2020-05-30T14:38:33 EDT" name="tearDown" finished-at="2020-05-30T14:38:38 EDT" duration-ms="5098" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.hrms.testcases.AddEmployeeValidationTest -->
      <class name="com.hrms.testcases.LoginPageValidationHW">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:37:32 EDT" name="setUp" finished-at="2020-05-30T14:37:34 EDT" duration-ms="2067" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="emptyPasswordValidation()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:37:34 EDT" name="emptyPasswordValidation" finished-at="2020-05-30T14:37:34 EDT" duration-ms="338" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Password cannot be empty expected:<Password cannot be empty> but was:<Text is not match>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Password cannot be empty expected:<Password cannot be empty> but was:<Text is not match>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:82)
at com.hrms.testcases.LoginPageValidationHW.emptyPasswordValidation(LoginPageValidationHW.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- emptyPasswordValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:37:34 EDT" name="tearDown" finished-at="2020-05-30T14:37:39 EDT" duration-ms="5100" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:37:39 EDT" name="setUp" finished-at="2020-05-30T14:37:41 EDT" duration-ms="1854" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="emptyUserNameValidation()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:37:41 EDT" name="emptyUserNameValidation" finished-at="2020-05-30T14:37:41 EDT" duration-ms="310" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Username cannot be empty expected:<Username cannot be empty> but was:<Text is not match>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Username cannot be empty expected:<Username cannot be empty> but was:<Text is not match>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:82)
at com.hrms.testcases.LoginPageValidationHW.emptyUserNameValidation(LoginPageValidationHW.java:40)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- emptyUserNameValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:37:41 EDT" name="tearDown" finished-at="2020-05-30T14:37:46 EDT" duration-ms="5098" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:37:46 EDT" name="setUp" finished-at="2020-05-30T14:37:48 EDT" duration-ms="1880" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="invalidPasswordValidation()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:37:48 EDT" name="invalidPasswordValidation" finished-at="2020-05-30T14:37:50 EDT" duration-ms="1674" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Invalid credentials expected:<Invalid credentials> but was:<Text is not matched>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Invalid credentials expected:<Invalid credentials> but was:<Text is not matched>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:82)
at com.hrms.testcases.LoginPageValidationHW.invalidPasswordValidation(LoginPageValidationHW.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidPasswordValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:37:50 EDT" name="tearDown" finished-at="2020-05-30T14:37:55 EDT" duration-ms="5096" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:37:55 EDT" name="setUp" finished-at="2020-05-30T14:37:58 EDT" duration-ms="3263" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginValidation()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:37:58 EDT" name="loginValidation" finished-at="2020-05-30T14:38:00 EDT" duration-ms="2096" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:38:00 EDT" name="tearDown" finished-at="2020-05-30T14:38:06 EDT" duration-ms="5105" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:38:06 EDT" name="setUp" finished-at="2020-05-30T14:38:08 EDT" duration-ms="2056" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="logoValidation()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:38:08 EDT" name="logoValidation" finished-at="2020-05-30T14:38:08 EDT" duration-ms="39" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:38:08 EDT" name="tearDown" finished-at="2020-05-30T14:38:13 EDT" duration-ms="5098" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:38:13 EDT" name="setUp" finished-at="2020-05-30T14:38:16 EDT" duration-ms="3070" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="titleValidation()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:38:16 EDT" name="titleValidation" finished-at="2020-05-30T14:38:16 EDT" duration-ms="7" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Human Management System expected:<Human Management System> but was:<Title is not found>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Human Management System expected:<Human Management System> but was:<Title is not found>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:82)
at com.hrms.testcases.LoginPageValidationHW.titleValidation(LoginPageValidationHW.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrms.testcases.LoginPageValidationHW@6a01e23]" started-at="2020-05-30T14:38:16 EDT" name="tearDown" finished-at="2020-05-30T14:38:21 EDT" duration-ms="5097" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.hrms.testcases.LoginPageValidationHW -->
      <class name="com.hrms.testcases.DashBoardPageValidationTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrms.testcases.DashBoardPageValidationTest@6293abcc]" started-at="2020-05-30T14:38:38 EDT" name="setUp" finished-at="2020-05-30T14:38:40 EDT" duration-ms="2582" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="dashBoardValidation()[pri:0, instance:com.hrms.testcases.DashBoardPageValidationTest@6293abcc]" started-at="2020-05-30T14:38:40 EDT" name="dashBoardValidation" finished-at="2020-05-30T14:38:49 EDT" duration-ms="8638" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dashBoardValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrms.testcases.DashBoardPageValidationTest@6293abcc]" started-at="2020-05-30T14:38:49 EDT" name="tearDown" finished-at="2020-05-30T14:38:54 EDT" duration-ms="5106" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.hrms.testcases.DashBoardPageValidationTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
